### PR TITLE
Automate sbt release steps

### PR DESCRIPTION
**Pull request by xerial**
This fix adds `publishSigned` and `sonatypeRelease` in `sbt release` steps.

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -55,6 +57,29 @@ object Build extends Build {
...
+        checkSnapshotDependencies,
+        inquireVersions,
+        runClean,
+        runTest,
+        setReleaseVersion,
+        commitReleaseVersion,
+        tagRelease,
+        ReleaseStep(
+          action = { state =>
+            val extracted = Project extract state
```
Comment (komamitsu) [inline project/Build.scala]: [Minor] Why don't you call `state.extract`?

Comment (xerial) [inline project/Build.scala]: state.extract is necessary to execute `runAggregated` task.

Comment (xerial) [inline project/Build.scala]: And we cannot call `state.extract`. We need to call `Project.extract(configuration key)`

**Comment by komamitsu** [comment]
Looks good.

**Pull request merged**